src/ape/api/accounts.py

```diff
@@ -164,24 +210,54 @@ class AccountContainerAPI:
     @property
     @abstractmethod
     def aliases(self) -> Iterator[str]:
-        ...
+        """
+        List all available aliases.
+
+        Returns:
+            Iterator[str]: List of aliases.
+        """
 
     @abstractmethod
     def __len__(self) -> int:
-        ...
+        """
+        Number of aliases.
+
+        """
 
     @abstractmethod
     def __iter__(self) -> Iterator[AccountAPI]:
-        ...
+        """
+        List all accounts.
+
+        Returns:
+            Iterator[AccountAPI] (``List[str]``)
+        """
 
     def __getitem__(self, address: AddressType) -> AccountAPI:
+        """
+        Raw input will be used to iterate and match the address.
+
+        Returns:
+            AccountAPI (:class:`~ape.api.accounts.AccountAPI`)
```

Comment:
The type has to come first actually
```suggestion
            :class:`~ape.api.accounts.AccountAPI`
```